src/data-catalog/create-dataset/CreateDataset.ts

```diff
@@ -100,11 +99,17 @@ export const CreateDataset = ({ storageSystem, storageSourceId }: CreateDatasetP
   });
 
   const errors =
-    validate({ storageSystem: storageSystemState, storageSourceId: storageSourceIdState, ...metadata }, constraints) ||
-    {};
+    validate(
+      {
+        storageSystem: storageSystemState,
+        storageSourceId: storageSourceIdState,
+        ...metadata,
+      },
+      constraints
+    ) || {};
   return h(FooterWrapper, {}, [
     loading && spinnerOverlay,
-    h(TopBar, { title: 'Create Dataset', href: '' }, []),
+    h(TopBar, { title: 'Upsert Dataset', href: '' }, []),
```

Comment:
At some point, it would be nice to separate out the Terra UI specific FooterWrapper and TopBar into page components (under src/pages) that wrap these feature components.

Reply:
I'm not upset at that, let me time box trying to do that and maybe explore it as part of this change.

Reply:
If this is a user facing title, I don't think it should use the word `Upsert`